common/src/main/java/org/keycloak/common/Profile.java

```diff
@@ -92,7 +92,12 @@ public enum Feature {
 
         DPOP("OAuth 2.0 Demonstrating Proof-of-Possession at the Application Layer", Type.PREVIEW),
 
-        LINKEDIN_OAUTH("LinkedIn Social Identity Provider based on OAuth", Type.DEPRECATED);
+        LINKEDIN_OAUTH("LinkedIn Social Identity Provider based on OAuth", Type.DEPRECATED),
+
+        DEVICE_FLOW("OAuth 2.0 Device Authorization Grant", Type.DEFAULT),
+
+        // Add new Feature above this line
+        ;
```

Comment:
I don't think we need to leave a comment for this, it should be pretty clear from the code itself where this goes.

Reply:
Thanks for the review. I removed the comment.